packages/data/src/test/registry.js

```diff
@@ -440,6 +440,22 @@ describe( 'createRegistry', () => {
 			expect( registry.select( 'reducer1' ).selector2() ).toEqual( 'result2' );
 			expect( selector2 ).toBeCalledWith( store.getState() );
 		} );
+
+		it( 'passes through optional context to getSelectors', () => {
+			const getSelectors = jest.fn();
+			const genericStore = {
```

Comment:
Same comment here re: `customStore` or something like that.

Reply:
Updated in f77e80c